content/contributing/opencue/Getting started/getting-started-linux.md

```diff
@@ -0,0 +1,331 @@
+---
+title: "Setting up a development environment on Linux"
+linkTitle: "Setting up a development environment on Linux"
+date: 2020-03-24
+weight: 1
+description: >
+  Set up your development environment on Linux
+---
+
+This is a guide to setting up a development environment on Linux that lets
+you run the entire OpenCue system, including Cuebot, CueGUI, and RQD. After
+you complete the setup, you can make changes to any part of the development
+environment.
+
+## Before you begin
+
+First, clone the
+[OpenCue git repository](https://github.com/AcademySoftwareFoundation/OpenCue)
+to your machine. You also need to download and install OpenCue's core
+dependencies:
+
+- [PostgreSQL](https://www.postgresql.org/download/linux/ubuntu/) version 9 or
+  greater.
+  
+  - We recommend installing Postgres using a `yum`-based Linux distribution, such as
+  [CentOS](https://www.centos.org/):
+
+   1. First, run `yum` to install the required Postgres packages:
+
+       ```shell
+       yum install postgresql-server postgresql-contrib
+       ```
+       
+   1. Next, initialize your Postgres installation and configure it to run as a
+    service:
+
+       ```shell
+       postgresql-setup initdb
+       systemctl enable postgresql.service
+       systemctl start postgresql.service
+       ```
+       
+   1. Create a superuser named after your current OS user, which is used for the
+    rest of the admin commands in this guide.
+
+       ```shell
+       su -c "createuser -s $USER" postgres
+       ```
+       
+- [Python 3.x](https://www.python.org/downloads/)
+
+  - We recommend installing Python 3 using a `yum`-based Linux distribution, such as [CentOS](https://www.centos.org/):
+
+       ```shell
+       yum install -y python36 python36-libs python36-devel python36-pip
+       ```
+   
+  - This will install Python 3.6.4 on your CentOS 7 machine as well as installing a native Python package management tool called pip. You can simply check it by  `python3.6 -V`.
+   
+- [Java SE JDK](https://www.oracle.com/technetwork/java/javase/downloads/index.html) version 11 or greater
+
+  - We recommend installing Java 11 using a `yum`-based Linux distribution, such as [CentOS](https://www.centos.org/):
+
+       ```shell
+       yum install java-11-openjdk-devel
+       ```
+   
+   - This will install openjdk version "11.0.3" on your CentOS 7 machine. You can simply check it by `java -version`.
+
+- It's also useful to install an IDEs. JetBrains has free community versions of the following options:
```

Comment:
I mentioned in my last comment that this is a *paragraph*, not part of the list. Please remove the list formatting (the preceding `-`).

Reply:
ok, sir